src/thing_list.c

```diff
@@ -3516,6 +3516,48 @@ struct Thing *get_nearest_object_at_position(MapSubtlCoord stl_x, MapSubtlCoord
   return _DK_get_nearest_object_at_position(stl_x, stl_y);
 }
 
+struct Thing *get_nearest_thing_at_position(MapSubtlCoord stl_x, MapSubtlCoord stl_y)
```

Comment:
How does this one differ from _DK_get_nearest_object_at_position?

Reply:
This returns any thing, whether it be an object, creature, corpse, effect, or whatever. _DK_get_nearest_object_at_position only returns an object.